# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/dns/nsIDNSByTypeRecord.idl
# Commit: 96bd98319c62
# Full Hash: 96bd98319c6242a9b05dc4723b928e88537424e1
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-26 03:24:37
# Regressor Bug: 1652667
# File Overlap Count: 1
# Description:
#   Bug 1652667 - P1: Add telemetry for when an HTTPSSVC record is received r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D87844
# ==============================================================================

diff -r b61a789151df -r 96bd98319c62 netwerk/dns/nsIDNSByTypeRecord.idl
--- a/netwerk/dns/nsIDNSByTypeRecord.idl	Tue Aug 25 17:29:21 2020 +0000
+++ b/netwerk/dns/nsIDNSByTypeRecord.idl	Tue Aug 25 17:29:32 2020 +0000
@@ -100,4 +100,8 @@
 {
   readonly attribute Array<nsISVCBRecord> records;
   nsISVCBRecord GetServiceModeRecord(in boolean aNoHttp2, in boolean aNoHttp3);
+  /**
+   * Returns true if one of SVCB records has IPv4 or IPv6 hint addresses.
+   */
+  readonly attribute boolean hasIPAddresses;
 };